pydy/codegen/ode_function_generators.py

```diff
@@ -885,11 +898,11 @@ def __init__(self, *args, **kwargs):
 
     def define_inputs(self):
 
-        if self.specifieds is None:
-            self.inputs = self.coordinates + self.speeds + self.constants
-        else:
-            self.inputs = (self.coordinates + self.speeds + self.specifieds
-                           + self.constants)
+        specifieds = []
+        if self.specifieds is not None:
+            specifieds = self.specifieds
+        self.inputs = chain(self.coordinates, self.speeds, specifieds,
```

Comment:
Can you wrap it in `list()` then, as it was before? I don't think efficiency really matters here.